api/enki.py

```diff
@@ -346,14 +355,14 @@ def checkout(self, patron, pin, licensepool, internal_format):
         )
         return loan
 
-    def loan_request(self, barcode, pin, book_id):
+    def loan_request(self, barcode, pin, book_id, enki_library_id):
```

Comment:
Both `checkout` and `fulfill` use this method. Checkout is borrowing the book, but fulfill is actually downloading it, correct?

Reply:
Yes, that's right. For Enki, checkout and fulfillment are the same API call, but we need slightly different methods to implement the `CirculationAPI` contract.